webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { TableVariant, TableText, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+
+import {
+  BugIcon,
+  SecurityIcon,
+  EnhancementIcon,
+} from '@patternfly/react-icons';
+
+import IsoDate from 'foremanReact/components/common/dates/IsoDate';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import PropTypes from 'prop-types';
+import TableWrapper from '../../../../components/Table/TableWrapper';
```

Comment:
The top bar of TableWrapper (the search and pagination) needs margin on the top and bottom to look more like the mockup. Right now it's very crowded. I'm thinking the solution should be in a `TableWrapper.scss` so we don't have to keep duplicating the CSS in every table..

Reply:
If we do this, that scss likely needs to be conditional. 
I have pages that use the table and expect little to no margin for example.
https://marvelapp.com/prototype/1121a8de/screen/72413287

Reply:
@parthaa I'm still seeing no margin between the tab bar and the TableWrapper top bar.  I agree with @Andrewgdewar that the styles should probably be per table.